data/role/jenkins-dev.yaml

```diff
@@ -9,6 +9,13 @@ psick::jenkins::jcasc::options_hash:
     Name: 'jenkins-slave-windows-dev'
     Role: 'jenkins-slave'
     Env: 'dev'
+  linux_slave_ami: 'ami-f61ab08c'
+  linux_slave_securityGroups: 'ssh-access, all-egress-dev, vpn-ingress-dev'
+  linux_slave_type: 'T2Medium'
```

Comment:
Another thing - wasn't this slave supposed to be a large number of cores to run builds in parallel?